README.md

```diff
@@ -50,6 +50,7 @@ Options include:
 ``` js
 {
   encrypted: true, // set to false to disable encryption if you are already piping through a encrypted stream
+  noise: true, // set to false to disable the NOISE handshake completely. This implies encrypted = false, and also disables the capability verification
```

Comment:
instead of implying encryption false here i think we should force to user to set both as you really need to know what you are doing when messing with this

Reply:
OK! Updated all three PRs.